public/locales/de/settings/general/search-new-tab.json

```diff
@@ -0,0 +1,3 @@
+{
+    "label": "Open search results in new tab"
```

Comment:
Don't add any files besides English. Crowdin will automatically detect, that the base language (English) was modified and add the necessary files itself.

Reply:
I removed this file and moved the label to settings/general/search-engine. Just for English, removed the rest.